File: src/main/scala/testBury/javaTest/Test2.java
package testBury.javaTest;

/**
 * @ClassName Test2
 * @Description TODO
 * @Author lenovo
 * @Date 2019/7/22 9:44
 **/
public class Test2 {
    public static void main(String[] args) {
        String sub="123";
        int[] intarry=new int[sub.length()];
        for(int i=0;i<sub.length();i++)
        {
            intarry[i]=Integer.parseInt(String.valueOf(sub.charAt(i)));
        }
        for(int i=0;i<intarry.length;i++)
        {
            System.out.println(intarry[i]);
        }
    }
}

File: src/main/scala/testBury/scalaTest/Test2.scala
package testBury.scalaTest

import org.apache.spark.{SparkConf, SparkContext}

/**
  * ClassName Test2
  * Description TODO
  * Author lenovo
  * Date 2019/10/10 10:22
  **/
object Test2 {
  def main(args: Array[String]): Unit = {
    val sparkConf = new SparkConf().setMaster("local[*]").setAppName("TTT")
    val context = new SparkContext(sparkConf)
    val aaa = context.textFile("e://aaa.txt")
    aaa.collect().foreach(line=> {
      println(line)
    })
  }
}
